lua/core/project.lua

```diff
@@ -1,50 +1,29 @@
-local M = {}
-
-function M.config()
-  lvim.builtin.project = {
+local M = {
+  defaults = {
     ---@usage set to false to disable project.nvim.
     --- This is on by default since it's currently the expected behavior.
     active = true,
 
     on_config_done = nil,
 
-    ---@usage set to true to disable setting the current-woriking directory
```

Comment:
Same as others, this will break autocomplete.

Reply:
I'm closing this as it's an obvious consequence of the task https://github.com/LunarVim/LunarVim/projects/1#card-67485846
I don't see a way for benefiting from the autocompletion while not predefining the whole structure, which should be avoided.